src/query/functions/fetch.go

```diff
@@ -86,15 +88,16 @@ func (o FetchOp) Node(controller *transform.Controller, storage storage.Storage,
 	}
 }
 
-// Execute runs the fetch node operation
-func (n *FetchNode) Execute(queryCtx *models.QueryContext) error {
+func (n *FetchNode) fetch(queryCtx *models.QueryContext) (block.Result, error) {
+	ctx := queryCtx.Ctx
+	sp, _ := opentracingutil.StartSpanFromContext(ctx, "fetch")
```

Comment:
Just curious why we're dropping the context from here? Shouldn't we update ctx to `sp, ctx := opentracingutil.StartSpanFromContext(ctx, "fetch")`?